### PR TITLE
Fixed two bugs in bibtex support and forward searching on windows with SumatraPDF

### PR DESCRIPTION
**Pull request by MaChenhao**
1. Fixed regexp in search of bibdata.
2. Fixed a bug in Tex_ForwardSearchLaTeX() when multiple tex file is used for SumatraPDF viewer on windows.

### CODE REVIEW

File: ftplugin/latex-suite/compiler.vim [hunk 1]
```diff
@@ -591,7 +591,7 @@ function! Tex_CompileMultipleTimes()
 		" The first time we see if we need to generate the bibliography and if the .bbl file
 		" changes, we will rerun latex.
 		" We use '\\bibdata' as a check for BibTeX and '\\abx' as a check for biber.
-		if runCount == 0 && Tex_IsPresentInFile('\v\\bibdata|\\abx', mainFileName_root.'.aux')
+		if runCount == 0 && Tex_IsPresentInFile('\\bibdata|\\abx', mainFileName_root.'.aux')
```
Comment (tyll) [inline ftplugin/latex-suite/compiler.vim]: Why is this change needed?

Comment (MaChenhao) [inline ftplugin/latex-suite/compiler.vim]: Because the bibtex cannot run automatically with the '\v' on my computer. It works well without it.
I think probably because I am working on windows and there is no vertical tab before the \bibdata line.

Comment (ixjlyons) [inline ftplugin/latex-suite/compiler.vim]: It seems to be related to #36

Comment (tyll) [inline ftplugin/latex-suite/compiler.vim]: ```\v``` is not a vertical tab but a modifier for the regular expression. How is the file called that your are editing where you need this patch to get it to work? What is the output of

    :echo Tex_GetMainFileName(':p:t:r')

when you run it in vim with the file open where you need this patch to work. Can you please verify that the filename you openend really shows this error?

Comment (MaChenhao) [inline ftplugin/latex-suite/compiler.vim]: When I am editing the file paper.tex in vim. The output is
```
:echo Tex_GetMainFileName(':p:t:r')
paper
```


I attached the paper.aux file which xelatex generated. There is a \bibdata in the file, but the regular expression cannot match with `\v`.
For example, when I type the following commands in vim, the outputs are different:
```
:echo Tex_IsPresentInFile('\\bibdata|\\abx', 'paper.aux')
1
:echo Tex_IsPresentInFile('\v\\bibdata|\\abx', 'paper.aux')
0
```

[paper.aux.txt](https://github.com/vim-latex/vim-latex/files/216478/paper.aux.txt)

**Comment by gerw** [comment]
@MaChenhao : I fixed the issue with the first commit. Can you rebase your second commit? Then it could be merged.
